docs/workflow/building/coreclr/cross-building.md

```diff
@@ -134,7 +136,7 @@ The following instructions assume you are on a Linux machine such as Ubuntu 14.0
 To build System.Private.CoreLib for Linux, run the following command:
 
 ```
-    lgs@ubuntu ~/git/coreclr/ $ build.sh arm debug verbose
+    lgs@ubuntu ~/git/runtime/ $ ./src/coreclr/build-runtime.sh arm debug verbose
```

Comment:
To build System.Private.CoreLib, you need to use the root build script now. `build-runtime.sh` doesn't build corelib.

Reply:
Updated. Is `--subsetCategory CoreClr` if `--subset CoreLib-NativeCoreLib` is specified?

Reply:
Updated. Is `--subsetCategory CoreClr` unnecessary if `--subset CoreLib-NativeCoreLib` is specified?

Reply:
As of now, yes. But it's better to say 
```
./build.(cmd/sh) --subsetCategory CoreClr --subset CoreLib-NativeCoreLib
```
Someday Mono could decide that CoreLib or NativeCoreLib are name of some subset they use. By specifying `subsetCategory` you specify the product you are focusing on.